README.md

```diff
@@ -8,6 +8,7 @@ This is a Kubernetes operator for [Grafana Tempo](https://github.com/grafana/tem
 * **Resource Limits**: Overall resource requests and limits can be specified in the TempoStack CR, and the operator will assign fractions of it to each component
 * **Authentication and Authorization**: Access to Tempo can be secured with RBAC
 * **Multitenancy**: Traces of multiple tenants can be stored in the same Tempo cluster
+* **Managed upgrades**: Updating the operator will automatically update all managed Tempo clusters
```

Comment:
Could you please add it as well to https://github.com/grafana/tempo/blob/main/docs/sources/tempo/setup/operator/_index.md#deploy-tempo-with-tempo-operator 

The https://github.com/grafana/tempo/blob/main/docs/sources/tempo/setup/operator/_index.md#deploy-tempo-with-tempo-operator has as well nicer wording for other features. It might be good to bring the content from the website here

Reply:
:+1: I synced the feature lists, will create the PR to the tempo repo after this PR is merged.